release-please-config.json

```diff
@@ -0,0 +1,39 @@
+{
+  "plugins": ["node-workspace"],
+  "bootstrap-sha": "78cc1e225bfc67300cfa9069fce257120fe92a6d",
+  "release-as": "21.4.0",
```

Comment:
this "global default" will apply to all the packages below for the next release that don't themselves override with a local `"release-as"` property.

Reply:
Nifty!